test/dynamo/test_base_hop.py

```diff
@@ -164,11 +247,8 @@ def f(inner, x, y):
         ):
             f(inner, x, y)
 
-        with self.assertRaisesRegex(
-            RuntimeError,
-            "Encountered input mutation during higher order op tracing for HOP",
-        ):
-            f(inner2, x, y)
+        # input mutation can be supported with auto_functionalize
```

Comment:
Why did this change?

Reply:
We need the change to test the input mutation schema generation. Can also delete the input mutation test and defer it to next PR where we add support.